pkgs/os-specific/linux/kernel/linux-zen.nix

```diff
@@ -12,12 +12,12 @@ buildLinux (args // {
     owner = "zen-kernel";
     repo = "zen-kernel";
     rev = "v${version}-zen1";
-    sha256 = "161bvrmic7gspkgkv0pqssk6dzv95vkxld69rir968khwlnpsnim";
+    sha256 = "1vn7rag28z6syl5xx0k69m2wl2k2y1ax622sb4g062f5sdpp5wd8";
   };
 
   extraMeta = {
-    branch = "5.8/master";
-    maintainers = with stdenv.lib.maintainers; [ atemu ];
+    branch = "5.9/master";
+    maintainers = with stdenv.lib.maintainers; [ atemu andresilva ];
```

Comment:
Nit: Should probably be done in a separate commit

Reply:
Done 👍